pkg/asset/installconfig/powervs/validation.go

```diff
@@ -183,3 +186,71 @@ func findSubnetInVPC(client API, subnets []string, region string, name string, p
 
 	return allErrs
 }
+
+// ValidateResourceGroup validates the resource group in our install config.
+func ValidateResourceGroup(client API, ic *types.InstallConfig) error {
+	ctx, cancel := context.WithTimeout(context.TODO(), 5*time.Minute)
+	defer cancel()
+
+	resourceGroups, err := client.ListResourceGroups(ctx)
+	if err != nil {
+		return fmt.Errorf("failed to list resourceGroups: %w", err)
+	}
+
+	switch ic.PowerVS.PowerVSResourceGroup {
+	case "":
+		return errors.New("platform:powervs:powervsresourcegroup is empty")
+	case "Default":
```

Comment:
I could see this not behaving as expected if a user has a resourcegroup named "Default" that isn't the default resourcegroup.. but that's quite the edge case

Reply:
Discussed this with @hamzy, I don't think we need to address it.